src/layouts/Dashboard/components/ReviewRequestModal/RequestOverview/RequestOverview.tsx

```diff
@@ -0,0 +1,335 @@
+  BiChevronRight,
+} from "react-icons/bi"
+import { TableVirtuoso } from "react-virtuoso"
+
+import { BxFileArchiveSolid } from "assets"
+
+interface DisplayedDateTime {
+  date: string
+  time: string
+}
+
+const getDateTimeFromUnixTime = (unixTime: number): DisplayedDateTime => {
+  const date = new Date(unixTime)
+  return {
+    date: date.toLocaleDateString("en-GB", {
+      month: "short",
+      year: "numeric",
+      day: "numeric",
+    }),
+    time: date.toLocaleTimeString("en-GB", {
+      timeStyle: "short",
+    }),
+  }
+}
+
+const getIcon = (iconTypes: EditedItemProps["type"]): JSX.Element => {
+  const iconType = iconTypes[0]
+  switch (iconType) {
+    case "nav": {
+      return (
+        <Icon ml="0.75rem" fontSize="1.25rem" fill="icon.alt" as={BiWorld} />
+      )
+    }
+    case "file": {
+      return (
+        <Icon
+          ml="0.75rem"
+          fontSize="1.25rem"
+          fill="icon.alt"
+          as={BxFileArchiveSolid}
+        />
+      )
+    }
+    case "setting": {
+      return <Icon ml="0.75rem" fontSize="1.25rem" fill="icon.alt" as={BiCog} />
+    }
+    case "page": {
+      return (
+        <Icon
+          ml="0.75rem"
+          fontSize="1.25rem"
+          fill="icon.alt"
+          as={BiFileBlank}
+        />
+      )
+    }
+    case "image": {
+      return (
+        <Icon ml="0.75rem" fontSize="1.25rem" fill="icon.alt" as={BiImage} />
+      )
+    }
+    default: {
+      // NOTE: This is done to ensure exhaustive type matching
+      const error: never = iconType
+      throw new Error(`Unmatched fileType: ${error}`)
+    }
+  }
+}
+
+type FileType = "page" | "nav" | "setting" | "file" | "image"
+
+export interface EditedItemProps {
+  type: FileType[]
+  name: string
+  path: string[]
+  url: string
+  lastEditedBy: string
+  lastEditedTime: number
+}
+
+const ItemName = ({ name, path }: Pick<EditedItemProps, "name" | "path">) => {
+  const theme = useTheme()
+  return (
+    <VStack align="flex-start">
+      <Text textStyle="subhead-2" textColor="text.label" noOfLines={1}>
+        {name}
+      </Text>
+      <Breadcrumb
+        as={Text}
+        textOverflow="ellipsis"
+        overflowX="auto"
+        spacing="2px"
+        separator={<BiChevronRight color={theme.colors.text.helper} />}
+      >
+        {path.map((item) => {
+          return (
+            <BreadcrumbItem>
+              <Text textStyle="caption-2" textColor="text.helper">
+                {item}
+              </Text>
+            </BreadcrumbItem>
+          )
+        })}
+      </Breadcrumb>
+    </VStack>
+  )
+}
+
+type LastEditedMetaProps = Pick<
+  EditedItemProps,
+  "lastEditedBy" | "lastEditedTime"
+>
+const LastEditedMeta = ({
+  lastEditedBy,
+  lastEditedTime,
+}: LastEditedMetaProps): JSX.Element => {
+  const { date, time } = getDateTimeFromUnixTime(lastEditedTime)
+  return (
+    <HStack>
+      <Avatar
+        bg="primary.100"
+        name={lastEditedBy.slice(0, 2).split("").join(" ")}
```

Comment:
I think this logic should be placed as some common function as I think it will also be used for the new navbar. Also to ensure that the way we get the initials is consistent throughout the CMS.

Reply:
sounds good! do you think that this will be used outside of the dashboard? if so, i'll probably place this in the root `utils` folder otherwise, jus this folder

Reply:
I know at least the new navbar will use this, the site dashboard will not be using this component.

Reply:
placed in `utils/text`!